tests/subsys/power/power_mgmt_soc/prj.conf

```diff
@@ -0,0 +1,11 @@
+# Enable config log
```

Comment:
where do you enable power management? are you assuming the boards have power management enabled?

Reply:
this ends up building only on the mec15xx boards, this is not a good enough coverage, those boards should not be enabling power management by default anyways, so we need another way to expand coverage where power management is supported.

Reply:
I used filter in testcase.yaml to filter out boards that do not have sleep residency. I will add CONFIG_SYS_POWER_MANAGEMENT=y in the prj.conf to enable power management. Will this still allow mec15xx boards only?

Reply:
Currently there are three boards that have SLEEP RESIDENCY specified and they should be covered by this test. mec15xxevb_assy6853, cc1352r1_launchxl and cc26x2r1_launchxl. I can only test the first one on my end but I don't have the other two available for testing.